arpeggio/__init__.py

```diff
@@ -1017,15 +1017,18 @@ class Terminal(ParseTreeNode):
             name in the case of an error node.
         suppress(bool): If True this terminal can be ignored in semantic
             analysis.
+        extra_value(object): additional information (e.g. the re matcher
```

Comment:
I would name this `extra_info` as `extra_value` wrongly implies it has something to do with `value`.

Reply:
name changed. Also adaptet textx branch...